### DISCUSSION TITLE
Newbie Questions - Prisma & Planetscale  Error P1001

### DISCUSSION THREAD

**Discussion by azarsenal** [planetscale/discussion#298] [Q&A]
Hello Everyone,

I am kicking the tires with PlanetScale and Prisma today.  I get an error when I execute the following command.  

```
npx prisma db push 

`
Environment variables loaded from ../../.env
Prisma schema loaded from schema.prisma
Datasource "db": MySQL database "xxxxx" at "us-east.connect.psdb.cloud:3306"

Error: P1001: Can't reach database server at `us-east.connect.psdb.cloud`:`3306`

Please make sure your database server is running at `us-east.connect.psdb.cloud`:`3306`.
```

I setup a database with no tables. The table has not been push to production.  

The error message is confusing.

What am I missing? 

Googling and Stackoverflow has not provided much insight. I hope by asking here, someone can help.

Thank you in advance.

**Comment by fooforge** [comment]
Hey @azarsenal,

Could you please share the contents of your `schema.prisma` file and the `DATABASE_URL` line from your `.env` file? Please make sure to omit any secrets or passwords.

**Reply by capt-amlan-padhi** [reply]
@fooforge  I too need some help with the same!. Can I share my files too??
